src/webui/www/private/downloadlimit.html

```diff
@@ -25,7 +25,7 @@
             'use strict';
 
             var hashes = new URI().getData('hashes').split('|');
-            setDlLimit = function() {
+            function setDlLimit() {
```

Comment:
To keep more in line with the original code, should we just add `var` in front of the function expression. e.g. `var setDlLimit = function() {`? Or is there a reason you're switching to function declarations.

Reply:
Actually I don't have a rational reason, using function declaration is more natural to me than function expression, or you think otherwise?

Reply:
I don't feel too strongly. The benefit of function expressions in this case is that we were already using them, but if switching to function declarations doesn't break anything then I suppose there's no harm.

Reply:
changed my mind, I probably shouldn't deviate from original style that much, so it is reverted.